docker/docker-compose.yaml

```diff
@@ -52,25 +52,26 @@ services:
       - ./workflow_data:/data
 
   # Celery Flower
-  celery-flower:
-    image: unstract/backend:${VERSION}
-    container_name: unstract-celery-flower
-    restart: unless-stopped
-    entrypoint: .venv/bin/celery
-    command: "-A backend flower --port=5555"
-    env_file:
-      - ../backend/.env
-    depends_on:
-      - execution-consumer
-      - redis
-    labels:
-      - traefik.enable=false
-    ports:
-      - "5555:5555"
-    environment:
-      - ENVIRONMENT=development
-    volumes:
-      - unstract_data:/data
+  # Uncomment locally if required for debugging celery.
+  # celery-flower:
+  #   image: unstract/backend:${VERSION}
+  #   container_name: unstract-celery-flower
+  #   restart: unless-stopped
+  #   entrypoint: .venv/bin/celery
+  #   command: "-A backend flower --port=5555 --purge_offline_workers=5"
+  #   env_file:
+  #     - ../backend/.env
+  #   depends_on:
+  #     - execution-consumer
+  #     - redis
+  #   labels:
+  #     - traefik.enable=false
+  #   ports:
+  #     - "5555:5555"
+  #   environment:
+  #     - ENVIRONMENT=development
+  #   volumes:
+  #     - unstract_data:/data
```

Comment:
@ritwik-g Instead of commenting we could keep it optional:

``` yaml
celery-flower:
    profiles:
      - optional
```

Reply:
Added profile debug